src/app/tab-preferences/tab-preferences.page.ts

```diff
@@ -140,21 +132,35 @@ export class Tab3Page {
     })
     this.notificationBase.disableToggleLock()
 
+    await this.updateUtils.convertOldToNewClientSettings()
```

Comment:
You can get rid of the await here, no need to wait for the completion of the migration on the preference screen (no need to increase loading time of this screen by waiting for it). By the time the user opens the client update page the small migration will be long done before that

Reply:
Good call, changed.